Sources/PostgresNIO/New/Data/Optional+PSQLCodable.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 13.01.21.
+//
+
+extension Optional: PSQLDecodable where Wrapped: PSQLDecodable {
```

Comment:
Ditch Xcode template headers.
```suggestion
extension Optional: PSQLDecodable where Wrapped: PSQLDecodable {
```